### ISSUE TITLE
Add Support for Nested Keras Models

### ISSUE THREAD

**Issue by kmader** [mil-tokyo/webdnn#285]
For doing more advanced networks (particularly UNET and ResNet blocks) it would be helpful to support ```keras.models.Graph``` in addition to ```keras.models.Sequential```

Current error message: 
```
convert(self, model, input_shapes)
    570             raise NotImplementedError("Non-sequential model is currently not implemented")
```

**Comment by milhidaka** [comment]
WebDNN already supports DAG model.
https://github.com/mil-tokyo/webdnn/blob/c56154709a95706a68343555f926222e1fc0516a/src/graph_transpiler/webdnn/graph/converters/keras.py#L567
ResNet-50 is the example. Please tell me what concrete model does not work.

**Comment by kmader** [comment]
Sorry for the misidentifying a problem

## Create the Model

```
import os
from keras.models import Model, Sequential
from keras.layers import Convolution2D
simple_model = Sequential()
simple_model.add(Convolution2D(1, kernel_size = (1,1), input_shape = (8, 8, 1)))
simple_model.save('simple_model.h5')

nested_model = Sequential()
nested_model.add(simple_model)
nested_model.add(Convolution2D(1, kernel_size = (1,1), input_shape = (8, 8, 1)))
nested_model.save('nested_model.h5')
```

## Convert the Models
```
import h5py
from webdnn.backend.interface.generator import generate_descriptor
from webdnn.graph.converters.keras import KerasGraphConverter
from webdnn.graph.graph import Graph
for c_model_path in ['simple_model.h5', 'nested_model.h5']:
    with h5py.File(c_model_path, "r") as model:
        converter = KerasGraphConverter()
        sys.stderr.write("Generating feedforward graph\n")
        input_shapes = [(1, 8, 8, 1)]
        graph = converter.convert(model, input_shapes)
```

## Output
```
Generating feedforward graph simple_model.h5
Generating feedforward graph nested_model.h5
---------------------------------------------------------------------------
NotImplementedError                       Traceback (most recent call last)
<ipython-input-5-1f682ac672ab> in <module>()
      8         sys.stderr.write("Generating feedforward graph {}\n".format(c_model_path))
      9         input_shapes = [(1, 8, 8, 1)]
---> 10         graph = converter.convert(model, input_shapes)

c:\Users\Kevin Mader\Dropbox\Informatics\webdnn\src\graph_transpiler\webdnn\graph\converters\keras.py in convert(self, model, input_shapes)
    570             raise NotImplementedError("Non-sequential model is currently not implemented")
    571 
--> 572         return converter.convert(input_shapes)

c:\Users\Kevin Mader\Dropbox\Informatics\webdnn\src\graph_transpiler\webdnn\graph\converters\keras.py in convert(self, input_shapes)
    468         current_vars = graph_inputs
    469         for layer in self.model_config["config"]:
--> 470             current_vars = self.convert_layer(layer["class_name"], layer["config"], current_vars)
    471         graph_outputs = current_vars
    472 

c:\Users\Kevin Mader\Dropbox\Informatics\webdnn\src\graph_transpiler\webdnn\graph\converters\keras.py in convert_layer(self, layer_class_name, layer_config, inputs)
     82             outputs = inputs
     83         else:
---> 84             raise NotImplementedError(f"Unknown Layer {layer_class_name}")
     85         return outputs
     86 

NotImplementedError: Unknown Layer Sequential
```

**Comment by milhidaka** [comment]
I understood the problem.
Current implementation does not consider nested model, and I will investigate if it is possible to support it in future release.

**Comment by milhidaka** [comment]
Released in v1.1.0.
Please try converting it again.
